### ISSUE TITLE
Implement folder settings

### ISSUE THREAD

**Issue by hiromipaw** [OpenArchive/Save-app-ios#243]
Settings for folders show active and archived folders depending on what the user has:

![All types](https://github.com/OpenArchive/Save-app-android/assets/1177650/28f6b633-f195-4aa2-b53c-baeb7e36c7cc)

![Active Projects Only](https://github.com/OpenArchive/Save-app-android/assets/1177650/33bc3d2e-1700-4c0f-858e-ec888380b43d)

![Active Projects Only-1](https://github.com/OpenArchive/Save-app-android/assets/1177650/ebf298a2-7b7c-43a4-8fb5-821cccb47b20)

You can pick name and license type for the full folder or all folders and you can also delete a folder:

![Projects-1](https://github.com/OpenArchive/Save-app-android/assets/1177650/fcc94989-2f48-4cac-a593-f8e82f991b5f)

![Projects](https://github.com/OpenArchive/Save-app-android/assets/1177650/644ca089-1779-411f-86e0-7a31c3d4a45d)

When a folder is deleted a confirmation popup is displayed:
![Remove Space](https://github.com/OpenArchive/Save-app-android/assets/1177650/94577695-1a4a-424c-b36f-ffb7a20d146c)

Remains to be clarified what should happen when a user changes the license for the whole server (aka. space)?

**Comment by tladesignz** [comment]
@purvi-ranawat, will be contained in version 3.0.0, build 81 currently uploading.

**Comment by purvi-ranawat** [comment]
@tladesignz : 

1. As mentioned in the requirement screenshot, the marked text is different and also, the title "Active folders'" is not displayed in the actual : 

Requirements : 



![Image](https://github.com/OpenArchive/Save-app-ios/assets/8190141/244fa964-1596-4a30-899a-25f3fe11d60e)


Actual : 



![Image](https://github.com/OpenArchive/Save-app-ios/assets/8190141/3495c16a-0a18-40b2-aa6e-0b0882fa62ad)


2. As mentioned in the requirement screenshot, the marked content is different and also , the title " Folder Name" is missing.

Requirements screenshot : 



![Image](https://github.com/OpenArchive/Save-app-ios/assets/8190141/2e356923-1827-424e-8146-3caa88796433)


Actual Screenshot : 



![Image](https://github.com/OpenArchive/Save-app-ios/assets/8190141/9bea37a9-ddd5-4147-8bdf-c2bbddba4504)

3. Delete confirmation pop up is different : 

Requirements screenshot : 



![Image](https://github.com/OpenArchive/Save-app-ios/assets/8190141/6cf79e60-2854-4f85-be9c-82d209b64886)


Actual screenshot : 



![Image](https://github.com/OpenArchive/Save-app-ios/assets/8190141/5062f73d-8df4-41e0-8be2-237de6323aa4)


Environment : 

OS : IOS 17.2
Device : Iphone XR
Build version : 3.0.0-build 83

**Comment by tladesignz** [comment]
Thank you, @purvi-ranawat, for your attention to detail, as always!

> @tladesignz :
> 
> 1. As mentioned in the requirement screenshot, the marked text is different and also, the title "Active folders'" is not displayed in the actual :

- I added the title.
- I didn't change the text of the CC stuff:
  - When doing the Android development, there was some spec change, because text was misleading otherwise in the first item.
  - The second item is the same on Android. Won't change to not impact translation again until management comes up with a good reason.

> 2. As mentioned in the requirement screenshot, the marked content is different and also , the title " Folder Name" is missing.

- I added the title.
- Regarding the different labels: See comment above.

> 3. Delete confirmation pop up is different :

The text was misleading (we're not deleting a "space" here, but a "folder"). Anyway, I adapted the text to get closer to the design template.


The fixes will be contained in version 3.0,0, build 85.
